### DISCUSSION TITLE
Not working in ubuntu 22.04  libssl

### DISCUSSION THREAD

**Discussion by Rxxbertx** [timschneeb/GalaxyBudsClient#316] [Q&A]
Hi, it gives me this error and idk why, i was trying to install all packages libssl but nothing...
![imagen](https://user-images.githubusercontent.com/104899410/166663048-e398b6cf-cd25-4ad7-8f86-15e3d96673fb.png)

**Comment by lucasrueda** [comment]
check this out:

https://gist.github.com/joulgs/c8a85bb462f48ffc2044dd878ecaa786
